modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/GlobalResources/GlobalResourceDto.cs

```diff
@@ -1,8 +1,13 @@
-﻿namespace Volo.CmsKit.Public.GlobalResources;
+﻿using System;
 
+namespace Volo.CmsKit.Public.GlobalResources;
+
+[Serializable]
 public class GlobalResourceDto
 {
     public string Name { get; set; }
-    
+
     public string Value { get; set; }
+
+    public DateTime? LastModificationTime { get; set; }
```

Comment:
We can use inheriting from `AuditedEntityDto` instead of defining `LastModificationTime` manually.

```diff
- public class GlobalResourceDto
+ public class GlobalResourceDto : AuditedEntityDto
{
    public string Name { get; set; }

    public string Value { get; set; }

-    public DateTime? LastModificationTime { get; set; }
}
```